scripts/get-external-data.py

```diff
@@ -227,15 +230,16 @@ def main():
                 else:
                     headers = {}
 
-                download = s.get(source["url"], headers=headers)
-                download.raise_for_status()
+                if not opts.local:
+                    download = s.get(source["url"], headers=headers)
+                    download.raise_for_status()
 
-                if (download.status_code == 200):
-                    if "Last-Modified" in download.headers:
+                if opts.local or (download.status_code == 200):
+                    if not opts.local and "Last-Modified" in download.headers:
                         new_last_modified = download.headers["Last-Modified"]
```

Comment:
If `ops.local` is set, `download` will be uninitialized. What will the lines below do?

Reply:
`opts.local` guards against utilizing `download`, the if conditional is read left to right and bails out early, never comparing `download.status`.

Conversely `not opts.local` is a check to allow using `download`, guarded above.